contracts/ERC677BridgeToken.sol

```diff
@@ -5,13 +5,17 @@ import "openzeppelin-solidity/contracts/token/ERC20/MintableToken.sol";
 import "openzeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol";
 import "./IBurnableMintableERC677Token.sol";
 import "./ERC677Receiver.sol";
+import "./libraries/Version.sol";
 
 
 contract ERC677BridgeToken is
     IBurnableMintableERC677Token,
     DetailedERC20,
     BurnableToken,
     MintableToken {
+
+    Version.Version public getBridgeInterfacesVersion = Version.Version(1, 0, 0);
```

Comment:
`getTokenInterfacesVersion` should be here.

Also, it seems that the code was changed significantly from the version we have in https://github.com/poanetwork/poa-bridge-contracts/releases/tag/1.0. So, lets announce the version `2.0.0` here.

Reply:
Updated name to `getTokenInterfacesVersion` and version to `2.0.0`